godtools/AnalyticsRelay.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AnalyticsRelay.swift
+//  godtools
+//
+//  Created by Greg Weiss on 3/26/18.
+//  Copyright © 2018 Cru. All rights reserved.
+//
+
+import Foundation
+
+class AnalyticsRelay {
+    
+    static let shared = AnalyticsRelay()
+    
+    var tractName: String = ""
+    var tractPlusCardName: String = ""
+    var tractCardCurrentNames: [String] = []
+    var tractCardNextNames: [String] = []
```

Comment:
I removed this property and the one above it and didn't encounter any issues, other that where the values are cleared and set. How are they used?

Reply:
I thought I would need the values here, but I think I can get values without these arrays. They've now been removed